package.json

```diff
@@ -91,6 +91,7 @@
     "vuex-class": "^0.3.2"
   },
   "devDependencies": {
+    "@4tw/cypress-drag-drop": "^2.2.4",
```

Comment:
this time, on the contrary, the `cypress-real-events` didn't work but the `@4tw/cypress-drag-drop` did

Reply:
So maybe we can use the first one for drag-n-drops, and the second one for arbitrary-delta move, or vice-versa (whichever works for the particular case)

Reply:
hum, that's interesting. Could you explain why do you say it didn't work?
I'd like to suggest we leave a note under an `.md` file in `e2e/` folder about this to guide test writer when to use cypress-drag-drop and when the cypress-real-events if we wanna keep both

Reply:
I have added `cypress/e2e/cypress-guidelines.md`. Does it look okay?